pkg/ovs/openflow/interfaces.go

```diff
@@ -403,3 +425,27 @@ type Packet struct {
 	ICMPEchoID      uint16
 	ICMPEchoSeq     uint16
 }
+
+// RegField specifies a register with a required bit range. regID is the register number, and rng is the range of bits
+// taken by the field. The OF client could use a RegField to cache or match varied value.
+type RegField struct {
+	regID int
+	rng   *Range
+	name  string
+}
+
+// RegMark uses a RegField to cache a fixed value or value enumeration. The RegMark is used to indicate the traffic
```

Comment:
How about: "RegMark is a value saved in a RegField"?

Do you feel RegFieldValue is easier to understand? I have no strong preference. Would let you decide.

Reply:
Ok.